### ISSUE TITLE
[Bug]: [Resizing] Vertical resizing not working.

### ISSUE THREAD

**Issue by prsidhu** [appsmithorg/appsmith#21482]

**Comment by Sripriya93** [comment]
Waiting on decision if we need vertical resizers or Auto height. QA on hold.

**Comment by Sripriya93** [comment]
Vertical resizing is disabled for Auto layout. Moving to done.
